docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,88 @@
+"""
+Coriolis parameters across the world
+====================================
+
+This code computes the Coriolis parameter for an area and stores it in a cube.
+
+It creates and plots a map showing the Coriolis parameters, overlaid on top of
+an Orthographic projection of the globe.
+
+"""
+
+import cartopy.crs as ccrs
+import iris
+from iris.coord_systems import GeogCS
+import iris.plot as iplt
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def main():
+    # Start with arrays for latitudes and longitudes, with a given resolution.
+    resolution = 200
+    longitudes = np.linspace(-180.0, 180.0, resolution)
+    latitudes = np.linspace(-90.0, 90.0, resolution)
+    lon2d, lat2d = np.meshgrid(longitudes, latitudes)
+
+    # Omega is the Earth's rotation rate, expressed in radians per second
+    omega = 7.29e-5
+
+    # Now to create our data
```

Comment:
Okay; I'm thinking of substituting in something like "We will create our example data using the 2D latitude array", so that it's clear to the user that they need the result from meshgrid and not the raw one-d latitudes.